### PR TITLE
fixed missing tables when using sqlcl with oracle

### PR DESCRIPTION
**Pull request by walkabout21**
Modified the schema.vim file to add support for sqlcl with oracle.  Fixed issue #172

### CODE REVIEW

**Comment by walkabout21** [comment]
I'm seeing a message that some checks were not successful.  Is there something else I need to do?

File: autoload/db_ui/schemas.vim [hunk 1]
```diff
@@ -1,3 +1,7 @@
+function! s:strip_quotes(results) abort 
+return split(substitute(join(a:results),'"','','g'))
```
Comment (kristijanhusak) [inline autoload/db_ui/schemas.vim:2]: Add indentation here

File: autoload/db_ui/schemas.vim [hunk 2]
```diff
@@ -106,7 +110,7 @@ let s:oracle_args = join(
       \    ";\n"
       \ ).';'
 let s:oracle_foreign_key_query = "
-      \SELECT DISTINCT RFRD.table_name, RFRD.column_name, RFRD.owner
+      \SELECT /*csv*/ DISTINCT RFRD.table_name, RFRD.column_name, RFRD.owner
```
Comment (kristijanhusak) [inline autoload/db_ui/schemas.vim:113]: Can you briefly explain why is this needed? It looks like a comment, but I'm assuming it has something to do with parsing.

**Review by kristijanhusak — CHANGES_REQUESTED** [review]
> I'm seeing a message that some checks were not successful. Is there something else I need to do?

Ignore that, tests are failing for some time now, I didn't had time to fix them.

File: autoload/db_ui/schemas.vim (hunk 1 above)
Comment (walkabout21) [reply autoload/db_ui/schemas.vim]: added indentation there and fixed indentation in dictionary update to match your two space standard.

File: autoload/db_ui/schemas.vim (hunk 2 above)
Comment (walkabout21) [reply autoload/db_ui/schemas.vim:113]: the /*csv*/ reads as a comment in sqlplus so it does nothing when executed by sqlplus.  In sqlcl (binary 'sql') it reads as a formatting hint so the output results are generated as comma delimited with " enclosures to make parsing easier.  I had to use the strip_quotes() function to strip the " enclosures generated by that format since there isn't a setting to remove the " enclosures.

Comment (kristijanhusak) [reply autoload/db_ui/schemas.vim:113]: Should same thing be set in `s:oracle_schemes_tables_query` or that somehow works like this?

Comment (walkabout21) [reply autoload/db_ui/schemas.vim:113]: It is set in s:oracle_schemes_tables_query and in the schemes query

Comment (kristijanhusak) [reply autoload/db_ui/schemas.vim:113]: I'm referring to this variable https://github.com/kristijanhusak/vim-dadbod-ui/pull/173/files#diff-64a7375523d30316a3830832410ac86411a390c17e61c5765399c08fd1614b34R121 . I don't see `csv` comment added to it, so I'm wondering if `schemes_tables_query` will work.

Comment (walkabout21) [reply autoload/db_ui/schemas.vim:113]: That was weird.  It was showing in my local but didn't push upstream.  I fixed and pushed a new commit.

Comment (walkabout21) [reply autoload/db_ui/schemas.vim:113]: I see what happened.  I was working in a separate test branch and when I squashed commits I missed that change.  I checked the diffs and everything is good now.

**Review by kristijanhusak — APPROVED** [review]

**Comment by kristijanhusak** [comment]
Looks good. I'm not able to test it at this moment, but I'll trust your judgment. thanks!
